src/handlers.pxi

```diff
@@ -689,7 +699,11 @@ async def fuse_setxattr_async (_Container c, name, value):
 
     # Make sure we know all the flags
     if c.flags & ~(libc_extra.XATTR_CREATE | libc_extra.XATTR_REPLACE):
-        raise ValueError('unknown flag(s): %o' % c.flags)
```

Comment:
This is fine. `fuse_setxattr_async` is not called directly through libfuse, so raising an exception here will be handled correctly.

Reply:
thanks, i removed that changeset again.